### PR TITLE
aws: add japanese translation 

### PR DESCRIPTION
**Pull request by uh-zz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ja/common/aws.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# aws
+
+> アマゾンウェブサービスの公式 CLI ツールです。
+> `aws S3`などいくつかのサブコマンドは、独自の使用方法に関するドキュメントが用意されています。
```
Comment (kbdharun) [inline pages.ja/common/aws.md:4]:
```suggestion
> `aws S3` のようないくつかのサブコマンドには、使用方法についての独自のドキュメントがあります。
```
This is what we use for Japanese pages. [Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md)

**Review by kbdharun — APPROVED** [review]
The translations LGTM. Thanks for your contribution. I have some suggestions for this page.

Edit: Seems like I pressed `Approve` instead of `Comment` 😳.

**Review by kbdharun — CHANGES_REQUESTED** [review]
Translate the placeholders and command arguments inside `{{}}` as it will be useful for native readers.

Tip: Leave the one in Line 25 as it is and translate the others.

File: pages.ja/common/aws.md (hunk 1 above)
Comment (uh-zz) [reply pages.ja/common/aws.md:4]: @kbdharun 
Thanks for the review!
It looks good to me, so I'll adopt it.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**